kombu/async/aws/sqs/message.py

```diff
@@ -19,7 +19,7 @@ class AsyncMessage(BaseAsyncMessage):
 
     def encode(self, value):
         """Encode/decode the value using Base64 encoding."""
-        return base64.b64encode(value).decode('utf-8')
+        return base64.b64encode(value.encode()).decode()
```

Comment:
Reading the code, I think that would actually be the opposite of what we want:

```
    def bytes_if_py2(s):
        """Convert str to bytes if running under Python 2."""
        return s
```

We want to input [`string` data if using Python 2](https://docs.python.org/2/library/base64.html#base64.b64encode) and to input [`bytes` data if using Python 3](https://docs.python.org/3.4/library/base64.html#base64.b64encode).

`.encode()` returns `string` in Python 2 and `bytes` in Python 3.  I'm pretty new to Python 3, so I may be missing something.  Maybe there's a better way to do this?